<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosqueInfo</class>
 <widget class="QDialog" name="MosqueInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeLabelText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Profile Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="MosqueNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mosque Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="OwnerLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Administrator:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MosqueNameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="OwnerProductEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNumberLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumberEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="EmailLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPush">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>-10</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Languages/build-MyMosqu-Desktop-Debug/Untitled-1.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IdFetch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Fetch</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Online_Activation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Online Activation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="Offline_Activation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Offline Activation </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Keyboard OnScreen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackPush">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Street">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="house_no">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EmailLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Street:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmailLabel_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>House Number : </string>
   </property>
  </widget>
  <widget class="QLabel" name="EmailLabel_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
